core/src/main/scala/weaponregex/WeaponRegeX.scala

```diff
@@ -1,10 +1,10 @@
 package weaponregex
 
 import weaponregex.parser.Parser
-import weaponregex.mutator.TreeMutator._
 import weaponregex.model.mutation._
 import weaponregex.mutator.BuiltinMutators
 import weaponregex.parser.{ParserFlavor, ParserFlavorJVM}
+import weaponregex.`extension`.RegexTreeExtension.RegexTreeMutator
```

Comment:
Why the backticks around `extension`? Is it a reserved keyword? If so, maybe rename it to `extensions` or something more related to the functionality you've created?

Reply:
At some point, the auto import imported it as `` `extension` ``. I just double checked, the auto import now imports it as `extension`, so I remove the backticks.